Maple2.Database/Storage/Game/GameStorage.Shop.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Maple2.Model.Game.Event;
+using Maple2.Model.Game.Shop;
+
+namespace Maple2.Database.Storage;
+
+public partial class GameStorage {
+    public partial class Request {
+        public Shop? GetShop(int shopId) {
+            return Context.Shop.FirstOrDefault(shop => shop.Id == shopId);
```

Comment:
`shop.Id` is the primary key so there can only be 1 result here. You should just use `Context.Shop.Find(shopId)`